File: docs/examples/enum-example-2.js
import { JunkFood, default as module, print } from './enum-example-2.zig';

console.log(`donut? ${module.junk === JunkFood.donut}`);
module.junk = JunkFood.taco; 
console.log(`taco? ${module.junk === JunkFood.taco}`);
print();
module.junk = 101;
console.log(`mystery food #100? ${module.junk === JunkFood(100)}`);
console.log(`mystery food #101? ${module.junk === JunkFood(101)}`);
print();

File: docs/examples/enum-example-2.zig
const std = @import("std");

pub const JunkFood = enum(u16) {
    hamburger,
    hotdog,
    donut,
    pizza,
    taco,
    _,
};

pub var junk: JunkFood = .donut;

pub fn print() void {
    std.debug.print("junk = {any}\n", .{junk});
}
